code/ZAS/Turf.dm

```diff
@@ -67,23 +67,20 @@
 
 				SSair.connect(sim, src)
 
-/turf/simulated/update_air_properties()
+/turf/simulated/update_air_properties(var/verbose = TRUE)
 	if(zone && zone.invalid)
 		c_copy_air()
 		zone = null //Easier than iterating through the list at the zone.
 
 	var/s_block = c_airblock(src)
-	if(s_block & AIR_BLOCKED)
+	if(s_block & AIR_BLOCKED && !locate(/obj/machinery/door/airlock) in src)
 		#ifdef ZASDBG
-		to_chat(if(verbose) world, "Self-blocked.")
```

Comment:
what the fuck why did this work to begin with

Reply:
#ifdef doesnt even look at the code inbetween if the thing isnt defined

Reply:
god damn